libr/core/cmd_type.c

```diff
@@ -340,11 +340,11 @@ static int cmd_type(void *data, const char *input) {
 				const char *ptr = input + 3;
 				ut64 addr = r_num_math (core->num, ptr);
 				r_anal_type_unlink (core->anal, addr);
-				} break;
+			} break;
 			}
 			break;
 		case '*':
-			sdb_foreach (core->anal->sdb_types,linklist,core);
+			sdb_foreach (core->anal->sdb_types, linklist, core);
```

Comment:
those indent fixes are good, but you should squash commits if one of them just "indents"